src/box/box.cc

```diff
@@ -232,7 +232,7 @@ box_check_writable(void)
 		error_set_uint(e, "queue_owner_id", id);
 		error_set_uint(e, "term", term);
 		error_append_msg(e, "synchro queue with term %llu belongs "
-				 "to %u", term, id);
+				 "to %u", (unsigned long long)term, id);
```

Comment:
I'd also cast `id` to `unsigned`, because `uint32_t` isn't guaranteed to be an alias for `unsigned`.

Reply:
Thanks, done.